Makefile

```diff
@@ -137,6 +137,9 @@ shfmt:
 	shfmt -ln bats -d -w tests/integration/*.bats
 	shfmt -ln bash -d -w man/*.sh script/* tests/*.sh tests/integration/*.bash
 
+gosec:
+	gosec -confidence medium -severity medium -exclude G301,G302,G304,G306,G307 -exclude-dir libcontainer/nsenter ./...
+
```

Comment:
So, it seems that you add `exclude-dir` here but not to golangci-lint gosec configuration. Why is that?

Reply:
Perhaps we should not have a separate make target for `gosec`, given that we already have a target to run golangci-lint (and we don't have separate targets for each of the linters it provides)

Reply:
Done